Neos.Fusion/Tests/Functional/FusionObjects/Fixtures/Fusion/DataStructure.fusion

```diff
@@ -86,3 +86,15 @@ dataStructure.advancedStartEndOrdering = Neos.Fusion:DataStructure {
 dataStructure.ignoreProperties < dataStructure.positionalOrdering {
 	@ignoreProperties = ${['f']}
 }
+
+dataStructure.nestingWithAndWithoutObjectName = Neos.Fusion:DataStructure {
+  keyWithoutType {
+    bar {
+      baz = 123
+    }
```

Comment:
I was curious whether this works recursively – and it does. But a test case for that would be nice:
```
Neos.Fusion:DataStructure {
  keyWithoutType {
    foo {
      bar {
        baz = 123
      }
    }
}
```

Reply:
Nevermind, the example _is_ already nested of course